src/scripts/Save.ts

```diff
@@ -184,20 +184,21 @@ class Save {
         $('#saveModal').modal('hide');
     }
 
-    // TODO (@Isha) reimplement
-    public static convertShinies(list: Array<string>) {
-        // let converted = [];
-        // for (let pokemon of list) {
-        //     let shiny = parseInt(pokemon['shiny']);
-        //     let name = pokemon['name'];
-        //     if (shiny == 1 && player.caughtShinyList.indexOf(name) == -1) {
-        //         player.caughtShinyList().push(pokemon['name']);
-        //         converted.push(pokemon['name']);
-        //     }
-        // }
-        // if (converted.length > 0) {
-        //     Notifier.notify("You have gained the following shinies: " + converted, GameConstants.NotificationOption.success)
-        // }
+    public static convertShinies(list: Array<any>) {
+        const converted = [];
+        list = list.filter(p => p.shiny);
+        for (const pokemon of list) {
```

Comment:
Does this work with the any type?

Reply:
Sure does, otherwise typescript throws an error expecting that everything in the array is a string.